WolvenKit/Views/Shell/DockingAdapter.xaml.cs

```diff
@@ -474,22 +474,23 @@ public void OnActiveProjectChanged()
                 if (!_hadLoadedProject && ItemsSource is ObservableCollection<IDockElement> oc)
                 {
                     _hadLoadedProject = true;
-                    //oc.Clear();
                 }
                 var layoutPath = Path.Combine(_viewModel.ActiveProject.ProjectDirectory, "layout.xml");
                 if (File.Exists(layoutPath))
                 {
-                    var reader = XmlReader.Create(layoutPath);
-                    var Debugging_A = PART_DockingManager.LoadDockState(reader);
-                    Trace.WriteLine(Debugging_A);
-                    reader.Close();
+                    Locator.Current.GetService<ILoggerService>().Info($"Trying to load project layout from {layoutPath}...");
+
+                    var loadedProjectLayout = PART_DockingManager.LoadDockState(layoutPath);
+                    
+                    Locator.Current.GetService<ILoggerService>().Info($"...Project layout load returned {loadedProjectLayout}, who knows what that means? Did it work?");
```

Comment:
> who knows what that means
It means, that the action was successfull...

Reply:
Fun fact: it doesn't

Reply:
Yes, it does. Just keep in mind, that it is only loading the state. 
So if the current "docked" windows are different then the previous saved, it will also return false.